api/v1/certificatepolicy_types.go

```diff
@@ -20,34 +23,28 @@ const (
 	Inform RemediationAction = "Inform"
 )
 
-// ComplianceState shows the state of enforcement
+// ComplianceState reports the observed status resulting from the definitions of the policy.
```

Comment:
Using Dale's suggestion
```suggestion
// CertificatePolicyStatus reports the observed status that results from parameters of the
// certificate policy.
```

Reply:
Since the `ComplianceState` field is directly under `CertificatePolicyStatus`, I'm thinking it could be tweaked a bit so that it's similar but not identical, something like this?
```
// ComplianceState reports the compliance that results from parameters of the certificate policy.
```